backend/requirements.in

```diff
@@ -1 +1 @@
-tfx==0.21.2
+tfx==0.21.4
```

Comment:
You'll need to regenerate requirements.txt too

Reply:
Just to clarify: Changing only the `requirements.in` will not affect any build behavior. You'll need to update `requirements.txt` by running `./update_requirements.sh <requirements.in >requirements.txt`